dbt_meshify/file_manager.py

```diff
@@ -67,14 +67,14 @@ def __init__(self,
 
     def read_file(self, path: os.PathLike) -> Union[Dict[str, str], str]:
         """Returns the yaml for a model in the dbt project's manifest"""
-        if path.endswith('.yml'):
+        if path.suffix == '.yml':
             return self.yml_file_manager.read_file(path)
         else:
             return self.base_file_manager.read_file(path)
```

Comment:
⚠️ **Slightly-irrelevant feedback**

This is outside the scope of your change, but here's a more succinct way of writing this in the future.

```python
if path.suffix == '.yml':
    return self.yml_file_manager.read_file(path)
return self.base_file_manager.read_file(path)
```